### PR TITLE
Change "Register" button text to "Register Name"

### PR DESCRIPTION
**Pull request by jamiedawsonyoung**
The actionText on the Register button should change from “Register” to “Register Name”

### CODE REVIEW

**Comment by jamiedawsonyoung** [comment]
[[dashboard] “Register” less confusing as “Register Name”](https://trello.com/c/7XgFEI7P/257-dashboard-register-less-confusing-as-register-name)

File: src/common/components/repository-row/index.js [hunk 1]
```diff
@@ -296,7 +296,7 @@ class RepositoryRow extends Component {
         actionSpinner = true;
         actionText = 'Checking...';
       } else {
-        actionText = 'Register';
+        actionText = 'Register Name';
```
Comment (bartaz) [inline src/common/components/repository-row/index.js]: Does "Name" needs to be in title case? I believe we are not using title case anywhere?

Comment (jamiedawsonyoung) [inline src/common/components/repository-row/index.js]: @matthewpaulthomas This one for you. I was just following what was in the card but I had the same thought as @bartaz

Comment (jamiedawsonyoung) [inline src/common/components/repository-row/index.js]: @bartaz Sorry, you're right. It's a lower-case n for "Register name"

**Comment by jamiedawsonyoung** [comment]
@bartaz I can't pull the trigger unfortunately… :-)

**Comment by bartaz** [comment]
@jamiedawsonyoung I'm not 100% sure, but now that you and @pmahnke are collaborators on this repo you should have rights to merge PRs to the master.

Don't you?

**Pull request merged**